GPU/Vulkan/TextureCacheVulkan.cpp

```diff
@@ -751,6 +751,15 @@ void TextureCacheVulkan::BuildTexture(TexCacheEntry *const entry) {
 	// such as when using replacement textures - but let's keep the same amount of levels.
 	int maxLevelToGenerate = maxLevel;
 
+	if (g_Config.iTexFiltering == TEX_FILTER_AUTO_MAX_QUALITY) {
+		// Boost the number of mipmaps.
+		int maxPossibleMipmaps = log2i(std::min(gstate.getTextureWidth(0), gstate.getTextureHeight(0)));
+		if (maxPossibleMipmaps != maxLevelToGenerate) {
+			maxLevelToGenerate = maxPossibleMipmaps;
+		}
```

Comment:
nit: the check is not needed :)

Reply:
Oh yeah, I only had it so I could set a breakpoint there, while debugging it.